lldb/source/Plugins/LanguageRuntime/Swift/SwiftLanguageRuntimeImpl.h

```diff
@@ -256,6 +256,26 @@ class SwiftLanguageRuntimeImpl {
     stripSignedPointer(swift::remote::RemoteAbsolutePointer pointer) = 0;
   };
 
+  /// A wrapper around TargetReflectionContext, which holds a lock to ensure
+  /// exclusive access.
+  struct ThreadSafeReflectionContext {
+    ThreadSafeReflectionContext(ReflectionContextInterface *reflection_ctx,
+                             std::recursive_mutex &mutex)
+        : m_reflection_ctx(reflection_ctx), m_lock(mutex) {}
+
```

Comment:
`ThreadSafeReflectionContext(ThreadSafeReflectionContext&) = delete` to avoid this being copied?

Reply:
It can't be copied or moved already because locks aren't copiable or movable.